test/test_helper.rb

```diff
@@ -223,6 +223,11 @@ def task_config(context: KubeclientHelper::TEST_CONTEXT, namespace: @namespace,
       KubernetesDeploy::TaskConfig.new(context, namespace, logger)
     end
 
+    def krane_black_box(command, args = "")
+      path = File.expand_path("../../exe/krane", __FILE__)
+      Open3.capture3("#{path} #{command} #{args}")
+    end
```

Comment:
With this addition, can we remove the similar methods from `test/exe/krane_test.rb` and `test/exe/version_test.rb`?

Reply:
Yep, apparently I copied/pasted not cut/pasted :(